documentation/Architecture/Overview.md

```diff
@@ -2,80 +2,105 @@
 
 This document outlines the architecture of the Microsoft Performance Toolkit SDK.
 
-For more detailed information on how to create your own project using the SDK, please view [Creating your own project](../Using-the-SDK/Creating-your-project.md). 
+For more detailed information on how to create your own project using the SDK, please view [Creating an SDK Plugin C# Project](../Using-the-SDK/Creating-your-project.md). 
 
-Examples:
-- [Custom Data Source Example](../../samples/SimpleDataSource/SimpleCustomDataSource.cs).
-- [Custom Data Processor Example](../../samples/SimpleDataSource/SimpleCustomDataProcessor.cs). 
 
+## High-Level Interfaces
 
-# Microsoft Performance Toolkit SDK
+The Microsoft Performance Toolkit SDK was built to empower users to analyze arbitrary data source (such as `.etl` files, `.ctf` files, SQL servers, etc). 
```

Comment:
I think people will get what you mean but data source such as SQL servers is a bit confusing to me. You mean server logs or data bases?

Reply:
Nope, I meant querying an actual SQL server :). I was trying to highlight that data sources need not be files. I think I agree that we don't need to explain this here, so I'll just change it to SQL logs